website/screens/components/inline/code/InlineCodePage.tsx

```diff
@@ -0,0 +1,105 @@
+import { DxcStack, DxcTable } from "@dxc-technology/halstack-react";
+import Code from "@/common/Code";
+import DocFooter from "@/common/DocFooter";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import Example from "@/common/example/Example";
+import basicUsage from "./examples/basicUsage";
+import alignment from "./examples/alignment";
+
+const sections = [
+  {
+    title: "Props",
+    content: (
+      <DxcTable>
+        <tr>
+          <th>Name</th>
+          <th>Default</th>
+          <th>Description</th>
```

Comment:
Change `th` from Description to `HeaderDescriptionCell`.

Reply:
Done!